includes/REST_API/Stories_Controller.php

```diff
@@ -57,6 +58,17 @@ class Stories_Controller extends Stories_Base_Controller {
 	 * @return WP_REST_Response Response object.
 	 */
 	public function prepare_item_for_response( $post, $request ) {
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+
+		// $_GET param is available when the response iss preloaded in edit-story.php
+		if ( isset( $_GET['web-stories-demo'] ) && 'edit' === $context && 'auto-draft' === $post->post_status ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Why not use `$request['web-stories-demo']` instead of `$_GET['web-stories-demo']`?